src/spdx3/model/annotation.py

```diff
@@ -32,12 +32,12 @@ class AnnotationType(Enum):
 @dataclass_with_properties
 class Annotation(Element):
     annotation_type: AnnotationType = None
-    subject: List[str] = field(default_factory=list)
+    subject: str = None
```

Comment:
`annotation_type` and `subject` are required fields that should not have a default value

Reply:
Agreed but `Annotation` is a child class of `Element` which already has optional fields. So we need to set default values here. But the implementation of the `__init__` function takes care of these fields and doesn't set default values for the  corresponding parameters.